docs/developer-guide/test-e2e.md

```diff
@@ -3,8 +3,19 @@
 The test [directory](https://github.com/argoproj/argo-cd/tree/master/test) contains E2E tests and test applications. The tests assume that Argo CD services are installed into `argocd-e2e` namespace or cluster in current context. A throw-away
 namespace `argocd-e2e***` is created prior to the execution of the tests. The throw-away namespace is used as a target namespace for test applications.
 
-The [/test/e2e/testdata](https://github.com/argoproj/argo-cd/tree/master/test/e2e/testdata) directory contains various Argo CD applications. Before test execution, the directory is copied into `/tmp/argocd-e2e***` temp directory and used in tests as a
-Git repository via file url: `file:///tmp/argocd-e2e***`.
+The [/test/e2e/testdata](https://github.com/argoproj/argo-cd/tree/master/test/e2e/testdata) directory contains various Argo CD applications. Before test execution, the directory is copied into `/tmp/argo-e2e***` temp directory and used in tests as a
+Git repository via file url: `file:///tmp/argo-e2e***`.
+
+!!! note "Rancher Desktop Volume Sharing"
```

Comment:
interesting, i use rancher and havent had any issues. are you using `virtiofs`?

Reply:
Nah, I'm using reverse-sshfs. I'm also using `make start-e2e-local`, which may be uniquely impacted.